src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -147,28 +148,27 @@ initialise1 as = initialise (head as) True <> initialise1 (tail as)
 
 -- Gate-level concepts
 buffer :: a -> a -> CircuitConcept a
-buffer a b = rise a ~> rise b <> fall a ~> fall b
+buffer a z = rise a ~> rise z <> fall a ~> fall z
```

Comment:
The reader of this code will not necessarily be familiar with the convention to use `a...` for inputs and `x...` for outputs. If you'd like to make the code easier to understand, I think it would be better to use `buffer i o`, `cElement i1 i2 o`, etc.